src/client/datascience/commands/notebookCommands.ts

```diff
@@ -52,6 +54,16 @@ export class NotebookCommands implements IDisposable {
         }
     }
 
+    private keybindSave() {
+        if (this.notebookEditorProvider.activeEditor) {
+            void this.commandManager.executeCommand('workbench.action.files.save', this.notebookEditorProvider.activeEditor.file);
```

Comment:
Little iffy on if we need to await here. In general given how we handle other VS Code commands I don't believe that we need to. We just send off the save or undo command to VS Code to handle and we don't care what it does or when it does it.

Reply:
Good catch.